shippo/tests/http_test.rs

```diff
@@ -0,0 +1,261 @@
+#![recursion_limit = "256"]
+
+use httpmock::MockServer;
+use reqwest::Url;
+
+use shippo::Shippo;
+
+mod data;
+
+#[tokio::test]
+async fn list_shipments() {
+    let server = MockServer::start();
+    let client = Shippo::new("token123");
+    let mock_url = Url::parse(&server.base_url()).unwrap();
+    let mock = server.mock(|when, then| {
+        when.method("GET")
+            .path("/shipments")
+            .header("authorization", String::from("ShippoToken token123")) // httpmock masks auth keys
```

Comment:
httpmock does not actually mask auth keys.  The request crate masks Authorization headers when displaying.